phantoms/MR_XCAT_qMRI/sim_ivim_sig.py

```diff
@@ -103,7 +105,7 @@ def contrast_curve_calc():
     D[36] = 3e-3  # 36 artery
     D[37] = 3e-3  # 37 vein
     D[40] = 1.31e-3  # 40 asc lower intestine : Hai-Jing et al. doi: 10.1097/RCT.0000000000000926
-    D[41] = 1.31e-3  # 41 trans lower intestine : Hai-Jing et al. doi: 10.1097/RCT.0000000000000926
+    D[41] = 1.31e-3  # 41 trans lower# If conversion fails, try splitting the input as a lis intestine : Hai-Jing et al. doi: 10.1097/RCT.0000000000000926
```

Comment:
Probably accidental add?

Reply:
Yeah, it is accidental. Will correct it.